content/docs/how-we-work/open-source-contribution/wordpress-contribution.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Wordpress Contribution
+weight: 3
+---
+
+# Wordpress Contribution
+
+## How to start Contributions
+
+### [Create a wordpress.org account](https://login.wordpress.org/register)
+
+Create a wordpress account to start contributing to WordPress and get help in the support forums.
+
+### Slack and Support Forums
+
+Join WordPress slack on [WordPress Chat](https://make.wordpress.org/chat/) and support forums on [WordPress.org documentation](https://wordpress.org/support/). Join important channels such as #Core, #Community, #Polyglots, #Support, #WPTV, #Documentation.
+
+### Check out useful tutorials and official links
+
+- WordPress Developer Resources: [Official WordPress Developer Resources](https://developer.wordpress.org/)
+
+- [Making WordPress](https://make.wordpress.org/)
+
+- [Five for the Future](https://make.wordpress.org/)
+
+### Finding Contribution Tasks
+
+- Custom plugins built while working on WordPress websites can be generalized and contributed.
+
+- Check out the Axelerant-supported plugins with all the information here to collaborate internally with team members. Some quick points to remember:
+
+  - Make sure you follow the [standard repository fork and merge request model](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) to contribute to the project.
+
+  - Check with the plugin creator to get write access to the repository or developer access on [WordPress.org](https://wordPress.org).
+
+  - Check out the internal release process documented [here](https://axelerant.atlassian.net/wiki/spaces/ASD/pages/1425899525/WordPress+Plugin+Release+Process) and the following repositories [1](https://github.com/michael-cannon/skel), [2](https://github.com/michael-cannon/skel-custom).
```

Comment:
These links are very outdated. The Confluence link is not even accessible. If the information in that Confluence page is useful and general, then we should bring it here in the handbook.